### PR TITLE
Feature/implement_transfer_operations

### PR DESCRIPTION
**Pull request by igroomgrim**

### CODE REVIEW

File: OmiseSwift/Transfer.swift [hunk 1]
```diff
@@ -46,3 +48,39 @@ public class Transfer: ResourceObject {
...
+
+
+
+
+
+
+
+
+
+
```
Comment (chakrit) [inline OmiseSwift/Transfer.swift]: @igroomgrim @igroomgrim @igroomgrim @igroomgrim @igroomgrim @igroomgrim @igroomgrim newlinesssssssssssssssssssssssssssssssssssssssssssssssssss

File: OmiseSwiftTests/TransferOperationsTest.swift [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            publicKey: "pkey_test_54flsro0dmplsfg80vm",
+            secretKey: "skey_test_54flpy4dc5jpkrmlpp6",
+            apiVersion: nil,
+            queue: (NSOperationQueue.currentQueue() ?? NSOperationQueue.mainQueue())
+        )
+        
+        return Client(config: config)
+    }
+    
+    func testTransferRetrive() {
```
Comment (chakrit) [inline OmiseSwiftTests/TransferOperationsTest.swift]: `Retrive`

File: OmiseSwift/Transfer.swift [hunk 3]
```diff
@@ -46,3 +48,28 @@ public class Transfer: ResourceObject {
         set { set("transaction", StringConverter.self, toValue: newValue) }
     }
 }
+
+public class TransferParams: Params {
+    public var amount: Int? {
```
Comment (chakrit) [inline OmiseSwift/Transfer.swift]: `amount` fields must be `Int64` krub.

Comment (igroomgrim) [inline OmiseSwift/Transfer.swift]: changed 👍

File: OmiseSwiftTests/TransferOperationsTest.swift (hunk 2 above)
Comment (igroomgrim) [inline OmiseSwiftTests/TransferOperationsTest.swift]: fixed 😆

**Pull request merged**